docs/Basics/02-working-with-files.md

```diff
@@ -12,17 +12,20 @@ tags: []
 
 ## What
 
-Studio supports 5 key API Design and Documentation files in multiple formats. The general categories for these files is located within the file creation dropdown (**+**). They include:
+Studio supports 5 key API Design and Documentation files in multiple formats. The general categories for these files are located within the file creation dropdown (**+**). They include:
 
-- **API**: API Description Documents
-- **Endpoint**: Individual Paths & Operations
-- **Model**: Reusable API Components
-- **Article**: Supplemental Markdown Files
-- **Image**: Supplemental Images
+- **API:** API Description Documents
+- **Endpoint:** Individual Paths & Operations
+- **Model:** Reusable API Components
+- **Article:** Supplemental Markdown Files
+- **Image:** Supplemental Images
 
-> There are also three other categories of files listed in the file creation dropdown: File (generic file creation), Folder (used for organizing content in your Git repository and Studio), & New Project (create a new Project)
+There are also three other categories of files listed in the file creation dropdown:
 
-> Folders are critical for accessing API Specification, Markdown, and Image files that aren’t formatted for Studio
```

Comment:
This sentence seemed confusing because how is an image or markdown "formatted for studio".